lib/view_model/contact_list/contact_list_view_model.dart

```diff
@@ -99,18 +99,22 @@ abstract class ContactListViewModelBase with Store {
 
   Future<void> delete(ContactRecord contact) async => contact.original.delete();
 
-  @computed
-  List<ContactRecord> get contactsToShow =>
-      contacts.where((element) => _isValidForCurrency(element)).toList();
+  ObservableList<ContactRecord> get contactsToShow => _currency != null ?
```

Comment:
shouldn't this be annotated with computed?

Reply:
Since I modified it to an observable list, I don’t think it needs to be computed.